projects/showcase/src/app/demo/examples/3d-buildings.component.ts

```diff
@@ -1,9 +1,12 @@
 import { Component } from '@angular/core';
 import { Map, SymbolLayerSpecification } from 'maplibre-gl';
+import { MapComponent, LayerComponent } from '@maplibre/ngx-maplibre-gl';
+import { MapTestingHelperDirective } from '../../helper/map-testing-helper.directive';
```

Comment:
What is this used for? Why is it not part of either the show case or ngx-maplibre?

Reply:
`MapTestingHelperDirective` and `MglMapResizeDirective` are part of the `demo.module.ts` imports. The schematic must have added this to each example component, because it was available to all examples through `demo.module.ts`.

Reply:
They should be loaded only when running the Cypress tests somehow...